tools/debian.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# THIS SHOULD RUN ON DEBIAN FROM THE PROJECT DIRECTORY
+
+# VARIABLES
+ARCH=x64
+DEB_SIGNATURE_ID=EF506BE8
+INPUT_TYPE=dir
+LICENSE=MIT
+NODE_VERSION=8.3.0 # Will upgrade after 1.6.5
+OUTPUT_TYPE=deb
+PACKAGE_NAME=logdna-agent
+S3_BUCKET=repo.logdna.com
+VERSION=$(cat ./package.json | grep version | cut -d'"' -f4)
+
+# PREPARE FOLDER AND FILES
+mkdir -p .build/
+mv tools/shared/before-remove tools/shared/after-upgrade tools/shared/init-script tools/shared/logrotate .build/
+
+# STEP 1: INSTALL DEPENDENCIES
+# STEP 1.1: INSTALL nexe
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+nvm install node
+nvm install ${NODE_VERSION}
+nvm use ${NODE_VERSION}
+sudo npm install -g nexe
+
+# STEP 1.2: INSTALL fpm and deb-s3
+sudo apt-get update -y
+sudo apt-get install -y ruby ruby-dev rubygems build-essential rpm
+sudo gem install --no-document fpm deb-s3
+
+# STEP 1.3: INSTALL ghr
+curl -O https://storage.googleapis.com/golang/go1.12.9.linux-amd64.tar.gz
+sha256sum go1.12.9.linux-amd64.tar.gz
+tar -xvf go1.12.9.linux-amd64.tar.gz
+sudo chown -R root:root ./go
+sudo mv go /usr/local
+echo "export GOPATH=$HOME/go" >> ~/.profile
```

Comment:
Should we use `$HOME` instead of `~`?  Might be a little more explicit

Reply:
@darinspivey, I think it doesn't matter as long as we keep it consistent since now I saw in some places I was using `$HOME` and in some `~` - changed all to `${HOME}`

Reply:
It doesn't really matter, but `$HOME` seems more explicit